src/components/addBook.css

```diff
@@ -0,0 +1,69 @@
+.Line {
+  width: 75rem;
+  height: 0.125rem;
+  margin: 2.5rem auto;
+  border: solid 1px var(--white);
+}
+
+.add-book-title {
+  width: 75rem;
+  height: 1.5rem;
+  display: block;
+  margin-top: 1.813rem;
+  margin-inline: auto;
+  margin-bottom: 0.813rem;
+  font-family: Montserrat, serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.add-book-form {
+  width: 75rem;
+  display: flex;
+  align-items: center;
+  margin: 0 auto;
+}
+
+.title-input {
+  width: 25rem;
+  height: 2.813rem;
+  margin: 1.188rem 1rem 0 0;
+  padding: 0.813rem 1.063rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+}
+
+.categories-select {
+  width: 15rem;
+  height: 2.813rem;
+  margin: 1.188rem 1rem 0 0;
+  padding: 0.813rem 1.063rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+}
+
+.author-input {
+  width: 22rem;
+  height: 2.813rem;
+  margin: 1.188rem 1rem 0 0;
+  padding: 0.813rem 1.063rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+}
+
+.add-book-btn {
+  width: 10rem;
+  height: 2.813rem;
+  margin: 1.188rem 0 0 0;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: var(--azure);
+}
```

Comment:
- Kindly, you didn’t stick the font, color, mergin, and padding of the given design, Please Update it to fill the requirement and have a better UI 👌
### Given design 👇
![2023-04-15_03-15](https://user-images.githubusercontent.com/52010789/232173764-fc171971-d19f-4989-9e90-8a9ead73a3cc.png)

### Your design 👇
![2023-04-15_03-22](https://user-images.githubusercontent.com/52010789/232173770-44116291-fe5d-4deb-aef0-337445914121.png)

Reply:
fixed